docs/changelog.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## 3.15.6 (2023-10-25)
+
+**✨ Improvements**
+
+- Define custom template sections for details pages #988
+
+**📝 Documentation**
+
+- ⚠️ The `enableMeteoWidge` in `global.json` is no longer supported, read the doc to see how to deactivate this widget on the details page
+
```

Comment:
```suggestion
## 3.15.6 (2023-10-25)

⚠️ Breaking change ⚠️ :  The `enableMeteoWidget` in `global.json` is no longer supported. By default the widget is activated on all instances. [Read the doc](https://github.com/GeotrekCE/Geotrek-rando-v3/blob/main/docs/customization.md#html--scripts) to see how to deactivate this widget on the details page

**✨ Improvements**

- Define custom template sections for details pages #988
```

Reply:
If we note this version as breaking change, we need up this version to `3.16.0`.

Reply:
True. While it is an important change because for those who disabled the forecast widget it will be reactivated, it not breaking. Maybe we should label it as a "major change" ? 

I'm not sure to well understand the versioning of Geotrek projects tbh, because if we use semantic versioning we should increment versions differently.

Reply:
ok a updated the whole thing as 3.16.0.